src/ApiService/ApiService/ServiceConfiguration.cs

```diff
@@ -82,6 +83,8 @@ public class ServiceConfiguration : IServiceConfig {
     public string? ApplicationInsightsAppId => GetEnv("APPINSIGHTS_APPID");
     public string? ApplicationInsightsInstrumentationKey => GetEnv("APPINSIGHTS_INSTRUMENTATIONKEY");
 
+    public string? AppConfigurationEndpoint => GetEnv("APPCONFIGURATION_ENDPOINT");
```

Comment:
If this is always required we should make it return `string?` and do an up-front check.

Ideally for the ServiceConfiguration I'd like us to have a method that loads everything from the environment up front so we catch any mis-configurations as soon as the program loads. But that's a future change :)